frontend/src/container/Publications.jsx

```diff
@@ -571,6 +619,20 @@ export class Publications extends React.Component {
             onClick={() => this.getPublicationPage(props.publicationPage.nextPageToken)}
           />
         )}
+        {state.showPublicationTitlePopup && (
+          <CardPopupCreateContent
+            onClose={this.handleTogglePublicationTitlePopup}
+            onValidate={this.handleClickValidatePublicationTitle}
+            label={this.props.t('New publication')}
+            customColor={COLORS.PUBLICATION}
+            faIcon='far fa-paper-plane'
+            contentName={this.state.newPublicationTitle !== undefined ? this.state.newPublicationTitle : ''}
+            onChangeContentName={this.handleChangeNewPublicationTitle}
+            btnValidateLabel={this.props.t('Publish')}
+            inputPlaceholder={this.props.t("Topic's subject")}
```

Comment:
you can remove the "this" in "this.props"
same for this.state

this.props and this.state are exploded at the beginning of the return

Reply:
done